cli/src/extension_api.ts

```diff
@@ -1,8 +1,24 @@
-// deno-lint-ignore-file ban-types
-
 /// <reference types="https://raw.githubusercontent.com/denoland/deno/v1.28.3/core/lib.deno_core.d.ts" />
 /// <reference lib="deno.window" />
 
+type Package = {
+  name: string;
+  version: string;
+};
+
+type Lockfile = {
+  package_type: string;
+  packages: Package[];
+};
+
+type ProcessOutput = {
+  stdout: string;
+  stderr: string;
+  success: boolean;
+  signal: number | null;
+  code: number | null;
```

Comment:
Before writing up these types, I hadn't thought about the fact that these values can be `null` (because they come from a rust `Option<i32>`)...